docs/getting-started-v2.md

```diff
@@ -0,0 +1,129 @@
+
+Voor test doeleinden is een mock van de RvIG variant van de BRP Persoon Bevragen API (BrpService) geïmplementeerd. Deze mock gebruikt de [testdataset persoonslijsten proefomgevingen GBA-V](https://www.rvig.nl/documenten/richtlijnen/2018/09/20/testdataset-persoonslijsten-proefomgevingen-gba-v) als input om de productie situatie zoveel mogelijk te kunnen simuleren.
+
+Ten behoeve van monitoring is Open Telemetry geïntegreerd in de BrpProxy. Met behulp van de docker-compose configuratie bestand wordt ook een [Jaeger](https://www.jaegertracing.io/) Container opgestart om de trace regels van de BrpProxy te visualiseren.
+
+Voor logging past de BrpProxy structured logging toe. Dit maakt het makkelijker voor tooling om de log regels te query-en en te filteren. Voor het visualiseren van de logging wordt [Seq](https://datalust.co/seq) gebruikt.
+
+In de volgende paragrafen is beschreven hoe de BrpProxy  t.b.v. test doeleinden op een lokale machine kan worden geïnstalleerd en geconfigureerd.
+
+### Prerequisites
+
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+- [git](https://git-scm.com/downloads)
+
+### Bouwen van de BrpProxy Container Image
+
+- Clone de Haal Centraal BRP Persoon bevragen repository als deze niet eerder al is ge-clone-d
+  ```sh
+  git clone https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen.git
+  ```
+- Ga naar de aangemaakte folder
+  ```sh
+  cd Haal-Centraal-BRP-bevragen
+  ```
+- Switch naar de `develop` branch
+  ```sh
+  git checkout develop
+  ```
+- Download de laatste wijzigingen/toevoegingen als de Haal Centraal BRP bevragen repository op een eerder moment is ge-clone-d
+  ```sh
+  git pull
+  ```
+- Bouw de BrpProxy Container Image. Dit kan enige tijd duren.
+  ```sh
+  docker-compose build
+  ```
+
+### Opstarten van de BrpProxy Container
+
+- Start de BrpProxy
+  ```sh
+  docker-compose up -d
+  ```
+- Roep de BRP Persoon Bevragen API aan met als base url: *http://localhost:5000*. Met behulp van curl kan op een snelle manier worden getest of de BrpProxy en de BrpService draaien. Start hiervoor een bash shell op en voer de volgende curl statement uit in de bash shell
+  ```sh
+  curl --location --request POST 'http://localhost:5000/haalcentraal/api/brp/personen' \
+  --header 'Content-Type: application/json' \
+  --data-raw '{
+      "type": "RaadpleegMetBurgerservicenummer",
+      "burgerservicenummer": ["999993653"],
+      "fields": ["burgerservicenummer"]
+  }'
+  ```
+- Optioneel kan postman worden gebruikt om de BrpProxy te testen. Importeer de `BRP-Bevragen-v2-postman-collection.json` postman collection onder de `test` map voor aanroep voorbeelden.  
+- Bekijk de log regels van de BrpProxy met behulp van Seq. Navigeer hiervoor naar: `http://localhost:5341`
+- Bekijk de trace regels van de BrpProxy met behulp van Jaeger. Navigeer hiervoor naar: `http://localhost:16686`
+
+### Stoppen van de BrpProxy Container
+
+- Stop de BrpProxy
+  ```sh
```

Comment:
staat sh hier expres op een eigen regel?

bij mij doet docker-compose down hetzelfde in de Command line en binnen sh, is sh dan wel nodig?